EssentialsDiscordLink/src/main/java/net/essentialsx/discordlink/AccountLinkManager.java

```diff
@@ -0,0 +1,110 @@
+package net.essentialsx.discordlink;
+
+import com.earth2me.essentials.IEssentialsModule;
+import net.ess3.api.IUser;
+import net.essentialsx.api.v2.services.discord.InteractionMember;
+import net.essentialsx.discord.JDADiscordService;
+
+import java.util.Map;
+import java.util.Optional;
+import java.util.Random;
+import java.util.UUID;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ThreadLocalRandom;
+
+public class AccountLinkManager implements IEssentialsModule {
```

Comment:
`IEssentialsModule` doesn't actually do anything as far as I'm aware.

Also, do we want to expose linked accounts through an API service? Might be useful to plugin devs who don't want to roll their own account linking or just want to integrate with our existing list of linked accounts.

Reply:
`module` is a field in EssentialsCommand, it will only take `IEssentialsModule`